python/paddle/fluid/parallel_executor.py

```diff
@@ -140,7 +131,7 @@ def __init__(self,
         # FIXME(zcd): is_distribution_ is a temporary field, because in pserver mode,
         # num_trainers is 1, so the current fields of build_strategy doesn't tell if
         # it's distributed model.
-        build_strategy.is_distribution = _is_pserver_mode(
+        build_strategy.is_distribution = framework.is_pserver_mode(
```

Comment:
Not sure is it appropriate that moving `is_pserver_mode` to `framework` model.
In fact, `parallel_executor.py` will be removed in the near future.